### PR TITLE
New quote component!

### PR DESCRIPTION
**Pull request by AWare**
new quote component- for comments etc

Remove blockquote--feature class

Get rid of the quote element from the patron page

New patron:
image
Old patron:
image

Old Quote on UK supporter:
image

New Quote on UK supporter:
image

New Quote on US supporter:
image

NEW EU:
image

### CODE REVIEW

File: frontend/assets/stylesheets/tools/_vars.scss [hunk 1]
```diff
@@ -54,13 +54,12 @@ $font-scale: (
     bodyHeading: (
         1: (font-size: 14, line-height: 20),
         2: (font-size: 16, line-height: 24),
-        3: (font-size: 18, line-height: 28),
-        4: (font-size: 20, line-height: 28),
```
Comment (joelochlann) [inline frontend/assets/stylesheets/tools/_vars.scss]: why delete this?

File: frontend/assets/stylesheets/components/_quote.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+.promo-quote__body {
+    @include clip-path('quote-mobile') {
+        padding-bottom: 168 - $gs-gutter * 2 - $gs-baseline;
+    }
+
+    @include clearfix();
+    background-color: color(neutral-7);
+    padding: $gs-baseline ($gs-gutter / 2) 0;
+    @include mq($from:tablet){
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_quote.scss]: Space before `{` here and elsewhere

File: frontend/assets/stylesheets/components/_quote.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        @include clip-path('quote') {
+            padding-bottom: 84 - $gs-gutter * 2 - $gs-baseline;
+        }
+    }
+
+}
+
+.promo-quote__text {
+    @include mq($from:tablet){
+    padding-left: 300px + 30px;
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_quote.scss]: indentation

File: frontend/assets/stylesheets/components/_quote.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    @include mq($from:tablet){
+        @include clip-path('quote') {
+            padding-bottom: 84 - $gs-gutter * 2 - $gs-baseline;
+        }
+    }
+
+}
+
+.promo-quote__text {
+    @include mq($from:tablet){
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_quote.scss]: you can just write `@include mq(tablet)` as shorthand for this, here and elsewhere

File: frontend/assets/stylesheets/components/_quote.scss [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    }
+}
+
+.promo-quote__cutout {
+    height:  gs-height(4);
+    clear: both;
+    position: absolute;
+    right: 30px + $gs-gutter;
+    bottom: -1.5em;
+    @include mq($from:tablet, $until:mem-full){
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_quote.scss]: just `@include mq(tablet)` will do here

File: frontend/assets/stylesheets/components/_quote.scss [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    padding-left: 300px + 30px;
+    }
+}
+
+.promo-quote__cutout {
+    height:  gs-height(4);
+    clear: both;
+    position: absolute;
+    right: 30px + $gs-gutter;
+    bottom: -1.5em;
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_quote.scss]: don't think this should be `em`

File: frontend/assets/stylesheets/components/_quote.scss (hunk 3 above)
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_quote.scss]: also, why `300px + 30px`? if 30px is a significant value (I can see it in a few other places) can you please extract into a meaningful variable name?

**Comment by joelochlann** [comment]
👍 💯 💎 thanks for putting up with my pedantry!

**Comment by AWare** [comment]
How else do we learn?

**Pull request merged**